examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -341,6 +343,11 @@ void emberAfThermostatClusterInitCallback(EndpointId endpoint)
     SetDefaultDelegate(endpoint, &delegate);
 }
 
+void emberAfActionsClusterInitCallback(EndpointId endpoint)
+{
+    Clusters::Actions::Instance::GetInstance()->SetDefaultDelegate(&sActionsDelegateImpl);
```

Comment:
This API surface makes no sense. If you want Actions clusters on two different endpoints, how is that supposed to work given this setup?

Reply:
Added support for multiple endpoints.